### PR TITLE
add iter8-trend doc

### PR DESCRIPTION
**Pull request by kalantar**
@Alan-Cha @sriumcp and I discussed rearranging the levels again. I think this is where we decided but it can always be moved later :)

### CODE REVIEW

File: content/iter8 trend/_index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+date: 2020-08-11T12:00:00+00:00
+menuTitle: iter8-trend
+title: Getting Started with iter8-trend
+weight: 25
+summary: Describes how to install and use iter8-trend
```
Comment (Alan-Cha) [inline content/iter8 trend/_index.md:6]: Just for note for the future, 

If we don't plan on auto generating our list files, then we don't need to fill out the `summary` front matter field anymore. Feel free to do so anyway if you'd like.

Comment (Alan-Cha) [reply content/iter8 trend/_index.md:6]: @kalantar @sriumcp

File: content/iter8 trend/_index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+date: 2020-08-11T12:00:00+00:00
+menuTitle: iter8-trend
+title: Getting Started with iter8-trend
+weight: 25
```
Comment (Alan-Cha) [inline content/iter8 trend/_index.md:5]: Is this the proper weight? Currently, this means that `iter8 trend` shows up between `Getting started` and `References`.

File: content/iter8 trend/_index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+This is invaluable to developers when tracking down a performance problem and
+pin-pointing it to a particular version that was deployed in the past.
+
+This is an optional component to [iter8](http://github.com/iter8-tools) and
+cannot run standalone. It should be installed either as part of iter8
+installation process or separately after iter8 is installed.
+
+The following short video introduces iter8-trend:
+
+[![Short Video](https://img.youtube.com/vi/FOtyqJPMj14/hqdefault.jpg)](https://youtu.be/FOtyqJPMj14)
```
Comment (Alan-Cha) [inline content/iter8 trend/_index.md:21]: @kalantar @huang195 Can someone replace this using Hugo's Youtube shortcode? 

See here: https://gohugo.io/content-management/shortcodes/#youtube

Comment (kalantar) [reply content/iter8 trend/_index.md]: done

File: content/iter8 trend/_index.md (hunk 2 above)
Comment (kalantar) [reply content/iter8 trend/_index.md:5]: Yes. We've thinking of rearranging things (again). @sriumcp will do most of the changes in his PR. I tried to make the minimal number of changes so as to not force him to refactor a lot.

Comment (Alan-Cha) [reply content/iter8 trend/_index.md:5]: Okay no problem!

**Pull request merged**